docs/examples/fields/test_example_8.py

```diff
@@ -0,0 +1,27 @@
+from dataclasses import dataclass, field
```

Comment:
please move this file into `docs/examples/decorators/test_example_1.py` and the docs into `docs/usage/decorators.rst`